onnxruntime-sys/build.rs

```diff
@@ -64,10 +64,19 @@ fn main() {
     let libort_install_dir = prepare_libort_dir();
 
     #[cfg(not(feature = "directml"))]
-    let (include_dir, lib_dir) = (
-        libort_install_dir.join("include"),
-        libort_install_dir.join("lib"),
-    );
+    let (include_dir, lib_dir) = match TRIPLET.os {
```

Comment:
directmlの場合はcfgで制御、androidの場合はmatchで制御、と不揃いになっているので、どちらかに統一するとわかりやすそうだなとちょっと思いました。
が、このPRとしてはこれで問題ないので、ちょっとFIXMEコメントだけお願いできると！
```suggestion
    // FIXME: directmlとandroidで処理の表現が違うので統一する
    let (include_dir, lib_dir) = match TRIPLET.os {
```